flocker/node/agents/test/blockdevicefactory.py

```diff
@@ -166,6 +166,9 @@ def _openstack(**config):
     a ``CinderBlockDeviceAPI``.  They will be configured to use the region
     where the server that is running this code is running.
 
+    # FLOC-1925 Make this a public helper function
+    # Move the REGION from environment parts elsewhere.
+
```

Comment:
So .. yes.  This is the refactoring I suggested earlier in the review I guess.  glad we're thinking the same thing. :smile: 

This should be public in a different module though.  `test` packages don't really get to have public interfaces.